open_schools_platform/student_management/student/views.py

```diff
@@ -5,33 +5,64 @@
 from rest_framework.response import Response
 
 from open_schools_platform.api.swagger_tags import SwaggerTags
+from open_schools_platform.common.utils import get_dict_excluding_fields
 from open_schools_platform.parent_management.families.selectors import get_family
 from open_schools_platform.parent_management.families.services import add_student_profile_to_family
-from open_schools_platform.student_management.student.serializers import StudentProfileSerializer
-from open_schools_platform.student_management.student.services import can_user_create_student_profile_check, \
-    create_student_profile
+from open_schools_platform.student_management.student.selectors import get_student_profile
+from open_schools_platform.student_management.student.serializers import StudentProfileCreateSerializer, \
+    StudentProfileUpdateSerializer, StudentProfileDataSerializer
+from open_schools_platform.student_management.student.services import can_user_interact_with_student_profile_check, \
+    create_student_profile, update_student_profile
 
 
 class StudentProfileApi(CreateAPIView):
     @swagger_auto_schema(
         operation_description="Creates Student profile via provided age, name and family id \n"
                               "Returns Student profile data",
-        request_body=StudentProfileSerializer,
-        responses={201: "Family was successfully created", 404: "There is no such family",
+        request_body=StudentProfileCreateSerializer,
+        responses={201: "Student profile was successfully created", 404: "There is no such family",
```

Comment:
I suppose write that
201: StudentProfileSerializer.
It is because front developers don't understand what they would get.